merkletree/tb.go

```diff
@@ -25,3 +34,17 @@ func (tb *TemporaryBinding) Serialize(strSig []byte) []byte {
 	tbBytes = append(tbBytes, tb.Value...)
 	return tbBytes
 }
+
+func (tb *TemporaryBinding) Verify(epoch uint64, ap *AuthenticationPath) bool {
```

Comment:
I think we should make it explicit that the epoch are verifying needs to come from an STR. Maybe we should be passing in an STR here instead of just an epoch (much like we're passing the whole auth path for the index and value comparison).